rasa_sdk/endpoint.py

```diff
@@ -90,7 +92,13 @@ async def health(_) -> HTTPResponse:
     @app.post("/webhook")
     async def webhook(request: Request) -> HTTPResponse:
         """Webhook to retrieve action calls."""
-        action_call = request.json
+        if request.headers.get("Content-Encoding") == "deflate":
```

Comment:
Are there other types of content-encoding that we're sending, or this is the only one?
I'd also extract `deflate` to a constant.

Reply:
This is the only one.